electron/package.json

```diff
@@ -34,7 +36,8 @@
     "dist": "yarn clear:ts && yarn build:ts",
     "postinstall": "copy-config",
     "prestart": "yarn dist",
-    "start": "electron . --no-sandbox"
+    "start": "electron . --no-sandbox",
+    "zip": "node bin/zip.js"
```

Comment:
Let's make use of `ts-node`.

Reply:
I think it's too much overhead to use ts-node and a separate `tsconfig.json` for two files. Additionally, I am already using `// @ts-check`.